src/main/webapp/app/course/manage/course-management-statistics.component.html

```diff
@@ -1,6 +1,18 @@
 <div>
     <div class="row mb-5">
         <h2 class="col-md-2" id="statistics-page-heading" jhiTranslate="statistics.course_statistics_title">Course statistics</h2>
+    </div>
+
+    <jhi-statistics-average-score-graph
+        *ngIf="courseStatistics"
+        [exerciseAverageScores]="courseStatistics.averageScoresOfExercises"
+        [courseAverage]="courseStatistics.averageScoreOfCourse"
+        [courseId]="courseId"
+    >
+    </jhi-statistics-average-score-graph>
+    <div style="border-top: 3px solid lightgray" class="w-100 mb-4"></div>
```

Comment:
Should we rather avoid inline styles here? Especially inline colors are bad when it comes to theming 😬

Reply:
You are right in general. But since there is no dedicated scss file for this component, creating one only for this separator seemed a bit disproportional for me. But if this would be convenient for theming, I will add it of course.